examples/zookeeper/config-and-run.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Check if conf folder is empty
+if [ ! "$(ls -A /opt/zookeeper/conf)" ]; then
```

Comment:
To test that `/opt/zookeper/conf` is a directory and empty, do:
```sh
if [[ -d '/opt/zookeeper/conf' && -z "$(ls -A '/opt/zookeeper/conf)" ]]; then
```

If you don't really care if it's a directory or not you could omit the first check.
```sh
if [[ -z "$(ls -A '/opt/zookeeper/conf)" ]]; then
```

Either way:
 - use a literal text check (`-z`) over the implicit check (`!`)
 - use `[[` over `[`